platform/kube/inject/inject.go

```diff
@@ -190,6 +193,24 @@ func GetInitializerConfig(kube kubernetes.Interface, namespace, injectConfigName
 		return nil, err
 	}
 
+	if c.IncludeNamespaces == nil && c.ExcludeNamespaces == nil {
+		return nil, fmt.Errorf("cannot leave both namespaces and excludeNamespaces")
```

Comment:
maybe reword it a bit to indicate what actions user needs to do, such as please either configure namespaces or excludeNamespaces.

Reply:
... or we default to `c.IncludeNamespaces = []string{v1.NamespaceAll}` if nothing is set.

Reply:
Yes, this seems more simple